Bottom: bd5b90d1e3159570e9c11c98b54213e89d611b36
Top:    ac513631fd49036caba05af4fdfe7a7c8178cafe
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 13:48:16 -0500

Update docker-entrypoint to call parent container's entrypoint

After updating the parent image, our entrypoint wasn't setting some environment
variables correctly. Using the parent's entrypoint script after our setup
fixes this problem.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index e1abd2b7..a5787c37 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -62,7 +62,7 @@
     - bacula_restores
   - name: Configure Docker networking
     role: docker-networking
-    net_connection: '{{ daedalus_net_connection }}'
+    network_connection: '{{ daedalus_net_connection }}'
     subnet: '{{ daedalus_subnet }}'
     services:
     - svn
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
diff --git a/docker/docker-mediawiki/docker-entrypoint.sh b/docker/docker-mediawiki/docker-entrypoint.sh
index 0e8fc7e2..ff93cb5c 100755
--- a/docker/docker-mediawiki/docker-entrypoint.sh
+++ b/docker/docker-mediawiki/docker-entrypoint.sh
@@ -108,10 +108,8 @@ case ${1} in
     wiki)
         # verify permissions
         chown -R www-data:www-data ${WIKI_BASE_DIR}
-        # Apache gets grumpy about PID files pre-existing
-        rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     LocalSettings)
